tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -240,54 +184,16 @@ def get_to_addrs_email_messages_map(self, sqs_message):
         to_addrs_to_resources_map = self.get_email_to_addrs_to_resources_map(sqs_message)
         to_addrs_to_mimetext_map = {}
         for to_addrs, resources in six.iteritems(to_addrs_to_resources_map):
-            to_addrs_to_mimetext_map[to_addrs] = self.get_mimetext_message(
+            to_addrs_to_mimetext_map[to_addrs] = get_mimetext_message(
+                self.config,
+                self.logger,
                 sqs_message,
                 resources,
                 list(to_addrs)
             )
         # eg: { ('milton@initech.com', 'peter@initech.com'): mimetext_message }
         return to_addrs_to_mimetext_map
 
-    def set_mimetext_headers(self, message, subject, from_addr, to_addrs, cc_addrs, priority):
-        """Sets headers on Mimetext message"""
-
-        message['Subject'] = subject
-        message['From'] = from_addr
-        message['To'] = ', '.join(to_addrs)
-        if cc_addrs:
-            message['Cc'] = ', '.join(cc_addrs)
-
-        if priority and self.priority_header_is_valid(priority):
-            priority = PRIORITIES[str(priority)].copy()
-            for key in priority:
-                message[key] = priority[key]
-
-        return message
-
-    def get_mimetext_message(self, sqs_message, resources, to_addrs):
-        body = get_rendered_jinja(
-            to_addrs, sqs_message, resources, self.logger,
-            'template', 'default', self.config['templates_folders'])
-
-        if not body:
```

Comment:
What is the reason for this?
I don't see a huge issue with an empty body email.. If user provides empty template imo it is ok to send empty email.

Reply:
guessing.. its probably not a good intent to send a user an empty email.